### ISSUE TITLE
Accessibility issue with alt tag

### ISSUE THREAD

**Issue by shadrach1999** [videojs/videojs.com#85]
All images conveying an information should have alt attributes to improve accessibility to the temporarily dissabled user

**Comment by gkatsev** [comment]
You're 100% right, we should make sure our site is accessible and not just the player.
Would you be able to do a quick pass? Otherwise, we'll get to it as soon as we can.

**Comment by shadrach1999** [comment]
yh sure
